lib/rubocop/cop/rails/prefer_fetch.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for access of app secrets and credentials
+      #
+      # @example
+      #   # bad
+      #   Rails.application.secrets.foo
+      #   Rails.application.credentials.bar
+      #
+      #   # good
+      #   Rails.application.secrets.fetch(:foo)
+      #   Rails.application.credentials.fetch(:bar)
+      #   Rails.application.credentials.fetch(:baz, "default_baz")
+      class PreferFetch < Cop
+        MSG = 'Prefer fetch over direct access.'
+
+        def_node_matcher :fetch_candidate?, <<-PATTERN
```

Comment:
It seems to be a pattern that shows that it is being directly accessed (`fetch` is not used).